src/main/java/net/rptools/maptool/client/functions/InputFunction.java

```diff
@@ -1272,10 +1274,20 @@ public void checkParameters(String functionName, List<Object> parameters)
   private ImageIcon getIcon(String id, int size, ImageObserver io) {
     // Extract the MD5Key from the URL
     if (id == null) return null;
```

Comment:
I know it was like this before you edited it, but if you don't mind can you change 
```
if (id == null) return null;
```
to 
```
if (id == null) {
  return null;
}
```
Just so it meets the coding standards and is consistent, and we don't have to create a separate PR to address that line y itself :)

Thanks